src/App/World/WorldStateSystem.cpp

```diff
@@ -97,3 +109,86 @@ void App::WorldStateSystem::ToggleVariant(Red::NodeRef aNodeRef, Red::CName aVar
 
     m_questPhaseExecutor->ExecuteNode(resetNode);
 }
+
+
+Red::Handle<App::CommunityWrapper> App::WorldStateSystem::GetCommunity(Red::NodeRef aNodeRef)
+{
+    if (!aNodeRef)
+        return {};
+
+    auto communityID = Red::ResolveNodeRef(aNodeRef);
+    if (!communityID)
+        return {};
+
+    auto wrapper = Red::MakeHandle<CommunityWrapper>();
```

Comment:
Add constructor that accepts community and `return Red::MakeHandle<CommunityWrapper>(communityWeak.Lock());`.